src/native/fs/fs_napi.cpp

```diff
@@ -2420,6 +2420,7 @@ fs_napi(Napi::Env env, Napi::Object exports)
         passwd_bufsize = 16384;
     }
     PASSWD_BUF_SIZE = passwd_bufsize;
+    ThreadScope::set_passwd_buf_size(PASSWD_BUF_SIZE);
```

Comment:
Maintaining the same in two places is messy for no good reason. Let choose one place. It can be the ThreadScope for ease of sharing it. Just add a getter and use it also from fs_napi.